src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -114,6 +116,17 @@ def _handle_global_configuration(config):
         enable_file_logging = prompt_y_n(MSG_PROMPT_FILE_LOGGING, default='n')
         allow_telemetry = prompt_y_n(MSG_PROMPT_TELEMETRY, default='y')
         answers['telemetry_prompt'] = allow_telemetry
+        cache_ttl = None
+        while not cache_ttl:
+            try:
+                cache_ttl = prompt(MSG_PROMPT_CACHE_TTL) or DEFAULT_CACHE_TTL
+                # ensure valid int by casting
+                cache_value = int(cache_ttl)
+                if cache_value < 1:
```

Comment:
That's what the `az cache delete` command already does. Yugang and I discussed having an `--ignore-cache` flag to do just this, but he felt `az cache delete` is sufficient.

Reply:
Also, you could supply a value of 0, but then the UX is really wonky because you can use --defer to save to the cache, but anything you try to do will say the cache is stale and retrieve from Azure. So a lot more work would have to be done to not add --defer is cache_ttl is 0 and so forth. If there end up being user voices to that effect, I'll be happy to do it. Until then, not so much.